pipelines/rhtas-operator-e2e.yaml

```diff
@@ -0,0 +1,260 @@
+              - name: instanceType
+                value: m5.large
+              - name: imageContentSources
+                value: |
+                  - source: registry.redhat.io/rhtas/rhtas-rhel9-operator
+                    mirrors:
+                      - quay.io/securesign/rhtas-operator
+                  - source: registry.redhat.io/rhtas/trillian-logsigner-rhel9
+                    mirrors:
+                    - quay.io/securesign/trillian-logsigner
+                  - source: registry.redhat.io/rhtas/trillian-logserver-rhel9
+                    mirrors:
+                    - quay.io/securesign/trillian-logserver
+                  - source: registry.redhat.io/rhtas/trillian-database-rhel9
+                    mirrors:
+                    - quay.io/securesign/trillian-database
+                  - source: registry.redhat.io/rhtas/fulcio-rhel9
+                    mirrors:
+                    - quay.io/securesign/fulcio-server
+                  - source: registry.redhat.io/rhtas/trillian-redis-rhel9
+                    mirrors:
+                    - quay.io/securesign/trillian-redis
+                  - source: registry.redhat.io/rhtas/rekor-server-rhel9
+                    mirrors:
+                    - quay.io/securesign/rekor-server
+                  - source: registry.redhat.io/rhtas/rekor-search-ui-rhel9
+                    mirrors:
+                    - quay.io/securesign/rekor-search-ui
+                  - source: registry.redhat.io/rhtas/rekor-backfill-redis-rhel9
+                    mirrors:
+                    - quay.io/securesign/trillian-redis
+                  - source: registry.redhat.io/rhtas/tuf-server-rhel9
+                    mirrors:
+                    - quay.io/securesign/scaffold-tuf-server
+                  - source: registry.redhat.io/rhtas/certificate-transparency-rhel9
+                    mirrors:
+                    - quay.io/securesign/certificate-transparency-go
+                  - source: registry.redhat.io/rhtas/client-server-cg-rhel9
+                    mirrors:
+                    - quay.io/securesign/cli-client-server-cg
+                  - source: registry.redhat.io/rhtas/client-server-re-rhel9
+                    mirrors:
+                    - quay.io/securesign/client-server-re
+                  - source: registry.redhat.io/rhtas/segment-reporting-rhel9
+                    mirrors:
+                    - quay.io/securesign/segment-backup-job
+                  - source: registry.redhat.io/rhtas/timestamp-authority-rhel9
+                    mirrors:
+                    - quay.io/securesign/timestamp-authority
+                  - source: registry.redhat.io/rhtas/trillian-createtree-rhel9
+                    mirrors:
+                    - quay.io/securesign/trillian-createtree
+                  - source: registry.redhat.io/rhtas/client-server-rhel9
+                    mirrors:
+                    - quay.io/securesign/client-server
```

Comment:
I may be misunderstanding here, but could we instead pass in the cluster version number using params? That way we could test each ocp version that is supported instead of testing ? wdyt? and please correct me if I am wrong :)

Although if I understand correctly, this would have us provision a few clusters at a time for a pr and pull so this might be way too much overhead, but just a passing comment

Reply:
yes, I think we can pass version as parameter :+1:  We would need to create separate test resource for each version (but that is fine IMHO).

The task time is about ~1h. Maybe we can test only on the latest version for every PR and test on multiple versions on push action. Does that make sense?

Another point is that this pipeline deploy newly created bundle. Does any operator update create new bundle image?

Reply:
Introduced new parameter `OCP_VERSION`